src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -176,6 +194,80 @@ export class TerminalTabbedView extends Disposable {
 		}
 	}
 
+	private _createChatTerminalsEntry(container: HTMLElement): void {
+		this._chatTerminalsEntry = dom.append(container, $('.terminal-tabs-chat-entry.monaco-list-row'));
+		this._chatTerminalsEntry.tabIndex = 0;
+		this._chatTerminalsEntry.setAttribute('role', 'button');
+
+		const entry = dom.append(this._chatTerminalsEntry, $('.terminal-tabs-entry'));
+		const icon = dom.append(entry, $('.terminal-tabs-chat-entry-icon'));
+		icon.classList.add(...ThemeIcon.asClassNameArray(Codicon.commentDiscussion));
+		this._chatTerminalsLabel = dom.append(entry, $('.terminal-tabs-chat-entry-label'));
+		this._chatTerminalsCount = dom.append(entry, $('.terminal-tabs-chat-entry-count'));
+
+		const runChatTerminalsCommand = () => {
+			void this._commandService.executeCommand('workbench.action.terminal.chat.viewChatTerminals');
+		};
+
+		this._register(dom.addDisposableListener(this._chatTerminalsEntry, dom.EventType.CLICK, e => {
+			e.preventDefault();
+			runChatTerminalsCommand();
+		}));
+		this._register(dom.addDisposableListener(this._chatTerminalsEntry, dom.EventType.KEY_DOWN, e => {
+			if (e.key === 'Enter' || e.key === ' ') {
+				e.preventDefault();
+				runChatTerminalsCommand();
+			}
+		}));
+	}
+
+	private _updateChatTerminalsEntry(): void {
+		if (!this._chatTerminalsEntry || !this._chatTerminalsLabel || !this._chatTerminalsCount) {
+			return;
+		}
+
+		const chatTerminalCount = this._terminalChatService.getToolSessionTerminalInstances().length;
+		const hasText = this._tabContainer.classList.contains('has-text');
+		const hasChatTerminals = chatTerminalCount > 0;
+
+		if (!hasChatTerminals) {
+			this._chatTerminalsEntry.style.display = 'none';
+			this._chatTerminalsLabel.textContent = '';
+			this._chatTerminalsCount.textContent = '';
+			this._chatTerminalsEntry.removeAttribute('aria-label');
+			this._chatTerminalsEntry.title = '';
+			this._chatTerminalsEntry.style.marginTop = '0px';
+			const widthWhenHidden = this._tabListElement.clientWidth;
+			if (widthWhenHidden > 0) {
+				this._layoutTabList(widthWhenHidden);
+			}
+			return;
+		}
+
+		this._chatTerminalsEntry.style.display = '';
+
+		if (hasText) {
+			this._chatTerminalsLabel.textContent = chatTerminalCount === 1
+				? localize('terminal.tabs.chatEntryLabelSingle', "{0} Chat Terminal", chatTerminalCount)
+				: localize('terminal.tabs.chatEntryLabelPlural', "{0} Chat Terminals", chatTerminalCount);
+			this._chatTerminalsCount.textContent = '';
+		} else {
+			this._chatTerminalsLabel.textContent = '';
+			this._chatTerminalsCount.textContent = `${chatTerminalCount}`;
+		}
+
+		const ariaLabel = chatTerminalCount === 1
+			? localize('terminal.tabs.chatEntryAriaLabelSingle', "Show 1 chat terminal")
+			: localize('terminal.tabs.chatEntryAriaLabelPlural', "Show {0} chat terminals", chatTerminalCount);
+		this._chatTerminalsEntry.setAttribute('aria-label', ariaLabel);
+		this._chatTerminalsEntry.title = ariaLabel;
```

Comment:
Should very rarely be using `title` anymore, it's not accessible and it would create a native hover which looks wrong now.